.github/lineage.yml

```diff
@@ -0,0 +1,6 @@
+---
+version: "1"
+
+lineage:
+  skeleton:
+    remote-url: https://github.com/cisagov/skeleton-generic.git
```

Comment:
Wrong skeleton I believe.

Reply:
Gah, you are correct!  Thanks for the sharp 👀.  Fixed in https://github.com/cisagov/ansible-role-docker/pull/16/commits/f2adc1693c993c022ae4e7206524985c41b7f794.

Reply:
I have nightmares about the wrong skeletons!  ☠️ 
Once a wrong skeleton bit me in the ass!  💀 🍑